garnett/trajectory.py

```diff
@@ -828,8 +872,8 @@ class Trajectory(BaseTrajectory):
     :param dtype: The default data type for trajectory data.
     """
 
-    TRAJ_ATTRIBUTES = ['N', 'type', 'types', 'type_ids', 'positions',
-                       'orientations', 'velocities', 'mass', 'charge',
+    TRAJ_ATTRIBUTES = ['N', 'type', 'types', 'type_ids', 'position',
```

Comment:
This is the list I was talking about earlier in the GSD reader. I would consider splitting this into two lists, one that has arrays like position/orientation/velocity/.../image and the other can have weird things like N, type, types, type_ids unless you can make it all work together.